### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by NAlexandrov** [cfsamson/powershell-script#10]
A new version throw an error during compilation:
```rust
error[E0433]: failed to resolve: could not find `windows` in `os`
  --> /Users/dos/.cargo/registry/src/github.com-1ecc6299db9ec823/powershell_script-0.3.1/src/lib.rs:63:14
   |
63 | use std::os::windows::process::CommandExt;
   |              ^^^^^^^ could not find `windows` in `os`

error[E0599]: no method named `creation_flags` found for mutable reference `&mut Command` in the current scope
  --> /Users/dos/.cargo/registry/src/github.com-1ecc6299db9ec823/powershell_script-0.3.1/src/lib.rs:84:56
   |
84 |             cmd.args(&["-NoProfile", "-Command", "-"]).creation_flags(CREATE_NO_WINDOW).spawn()?
   |                                                        ^^^^^^^^^^^^^^ method not found in `&mut Command
```

**Comment by NAlexandrov** [comment]
As a temporary workaround I created target specific functions:

```rust
#[cfg(target_family = "windows")]
fn get_process(mut cmd: Command) -> io::Result<Child> {
    use std::os::windows::process::CommandExt;
    let os = std::env::consts::OS;
    const CREATE_NO_WINDOW: u32 = 0x08000000;

    cmd.args(&["-NoProfile", "-Command", "-"])
        .creation_flags(CREATE_NO_WINDOW)
        .spawn()
}

#[cfg(target_family = "unix")]
fn get_process(mut cmd: Command) -> io::Result<Child> {
    cmd.args(&["-NoProfile", "-Command", "-"]).spawn()
}
```

Maybe you can do something similar?

**Comment by cfsamson** [comment]
Thanks for reporting. This seems to be caused by the latest changes in #9. I think the cleanest way to fix it is what you suggested. Updated and published the fix to cargo as version 0.3.2.
